packages/synapse-react-client/src/components/dataaccess/AccessRequirementDashboard.integration.test.tsx

```diff
@@ -7,26 +7,27 @@ import selectEvent from 'react-select-event'
 import {
   AccessRequirementDashboard,
   AccessRequirementDashboardProps,
+  AR_NAME_OR_ID_SEARCH_PARAM_KEY,
+  AR_TYPE_SEARCH_PARAM_KEY,
+  RELATED_PROJECT_ID_SEARCH_PARAM_KEY,
+  REVIEWER_ID_SEARCH_PARAM_KEY,
 } from './AccessRequirementDashboard'
 import { createWrapper } from '../../testutils/TestingLibraryUtils'
 import { server } from '../../mocks/msw/server'
 import {
   MOCK_USER_ID,
   MOCK_USER_NAME,
 } from '../../mocks/user/mock_user_profile'
-import { AccessRequirementTable } from './AccessRequirementTable'
+import SynapseClient from '../../synapse-client'
+import { MOCK_ACCESS_TOKEN } from '../../mocks/MockSynapseContext'
 
-const AR_TABLE_TEST_ID = 'AccessRequirementTableTestId'
 const NAME_CONTAINS_PREFIX = 'abc'
 const RELATED_PROJECT_ID = 'syn123'
 
-jest.mock('../../../src/components/dataaccess/AccessRequirementTable', () => ({
-  AccessRequirementTable: jest.fn().mockImplementation(() => {
-    return <div data-testid={AR_TABLE_TEST_ID}></div>
-  }),
-}))
```

Comment:
Remove mocked table and update tests to 
1. verify that a table is displayed
2. verify the expected API call is made